forum/users/validators.py

```diff
@@ -0,0 +1,22 @@
+from django.core.exceptions import ValidationError
+from rest_framework import serializers
+
+
+class SpecialCharacterValidator:
+    def validate(self, password, user=None):
+        if not any(char in "!@#$%^&*()_+" for char in password):
```

Comment:
The special characters list is hardcoded, which limits flexibility. Allow the character set to be customizable via a class attribute:
```
class SpecialCharacterValidator:
    special_characters = "!@#$%^&*()_+"

    def validate(self, password, user=None):
        if not any(char in self.special_characters for char in password):
            raise ValidationError(
                f"The password must contain at least one special character ({self.special_characters})."
            )
```